src/components/loading/loading-indicador.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { LoadingContainer } from './loading-styles';
+
+export const LoadingIndicator = () => {
+  return (
```

Comment:
nit: como essa função não faz nenhuma lógica e apenas retorna um componente vc pode retornar de maneira implicita.
```suggestion
export const LoadingIndicator = () => (
```